crates/rustrace/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rustrace"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { version = "4.2.4", features = ["derive"] }
+fork = "0.1.21"
+nix = "*"
+anyhow = "*"
```

Comment:
@goodhoko started using `eyre` in #12, I suggest following suit here.

Reply:
yup, though I can switch to anyhow easily if we wan to be consistent with other Tonari projects that all seem to use anyhow.

I don't have any good reason to prefer eyre over anyhow.

Reply:
Colors? ;)